### ISSUE TITLE
Infinite loop when Access Files Already exist

### ISSUE THREAD

**Issue by jfarbowitz** [mediamicroservices/mm#38]
I tried to use ingestfile, but already had access files in the same directory as the video file. The program is continually displaying this error message and not doing anything else

```
 2015-12-16T14:50:45 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:45 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
  2015-12-16T14:50:46 - WARNING /Users/archivesx/Desktop/UMA00794/objects/access/mp3/UMA00794.mp3 already exists, skipping transcode
```

**Comment by dinahhandel** [comment]
oh no! weird. Thanks for pointing this out @jfarbowitz !! I'll investigate next week.

**Comment by dinahhandel** [comment]
This should be fixed with  8370ad9 Thanks again @jfarbowitz for pointing this out!
